test/views/declaration/DepartureTransportViewSpec.scala

```diff
@@ -91,83 +86,132 @@ class DepartureTransportViewSpec extends UnitViewSpec with CommonMessages with S
-          section
-            .getElementById("Departure_WagonNumber-label")
+        "have 'Wagon number' option" in {
+          view.getElementById("Departure_WagonNumber").attr("value") mustBe WagonNumber
+          view
+            .getElementsByAttributeValue("for", "Departure_WagonNumber")
             .text() mustBe "declaration.transportInformation.meansOfTransport.wagonNumber"
         }
 
-        "have 'Vehice number' option" in {
-          section
-            .getElementById("Departure_VehicleRegistrationNumber-label")
+        "have 'Vehicle number' option" in {
+          view.getElementById("Departure_VehicleRegistrationNumber").attr("value") mustBe VehicleRegistrationNumber
+          view
+            .getElementsByAttributeValue("for", "Departure_VehicleRegistrationNumber")
             .text() mustBe "declaration.transportInformation.meansOfTransport.vehicleRegistrationNumber"
         }
 
         "have 'flight number' option" in {
-          section
-            .getElementById("Departure_IATAFlightNumber-label")
+          view.getElementById("Departure_IATAFlightNumber").attr("value") mustBe IATAFlightNumber
+          view
+            .getElementsByAttributeValue("for", "Departure_IATAFlightNumber")
             .text() mustBe "declaration.transportInformation.meansOfTransport.IATAFlightNumber"
         }
 
         "have 'aircraft registration' option" in {
-          section
-            .getElementById("Departure_AircraftRegistrationNumber-label")
+          view.getElementById("Departure_AircraftRegistrationNumber").attr("value") mustBe AircraftRegistrationNumber
+          view
+            .getElementsByAttributeValue("for", "Departure_AircraftRegistrationNumber")
             .text() mustBe "declaration.transportInformation.meansOfTransport.aircraftRegistrationNumber"
         }
 
-        "have 'eni code' optopn" in {
-          section
-            .getElementById("Departure_EuropeanVesselIDNumber-label")
+        "have 'european vessel id' option" in {
+          view.getElementById("Departure_EuropeanVesselIDNumber").attr("value") mustBe EuropeanVesselIDNumber
+          view
+            .getElementsByAttributeValue("for", "Departure_EuropeanVesselIDNumber")
             .text() mustBe "declaration.transportInformation.meansOfTransport.europeanVesselIDNumber"
         }
 
         "have 'inland waterway' option" in {
-          section
-            .getElementById("Departure_NameOfInlandWaterwayVessel-label")
+          view.getElementById("Departure_NameOfInlandWaterwayVessel").attr("value") mustBe NameOfInlandWaterwayVessel
+          view
+            .getElementsByAttributeValue("for", "Departure_NameOfInlandWaterwayVessel")
             .text() mustBe "declaration.transportInformation.meansOfTransport.nameOfInlandWaterwayVessel"
         }
       }
 
-      "display 'Reference' section" which {
-        "have label" in {
+      "display 'Transport details type' id number section " which {
```

Comment:
Could this be tested together with the radio section above? I mean each case with matching case above.

Reply:
I thought that would be testing too much at one time?

Reply:
But radio option and its reveal component could be treated as a single "thing" ? You cannot have the revealing component with the radio option.

Reply:
Merged the two sets of tests together